public/app/plugins/datasource/elasticsearch/configuration/DerivedFields.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { css } from 'emotion';
+import { Button, DataLinkBuiltInVars, stylesFactory, useTheme, VariableOrigin } from '@grafana/ui';
+import { GrafanaTheme } from '@grafana/data';
+import { DerivedFieldConfig } from '../types';
+import { DerivedField } from './DerivedField';
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => ({
+  infoText: css`
+    padding-bottom: ${theme.spacing.md};
+    color: ${theme.colors.textWeak};
+  `,
+  derivedField: css`
+    margin-bottom: ${theme.spacing.sm};
+  `,
+}));
+
+type Props = {
+  value?: DerivedFieldConfig[];
+  onChange: (value: DerivedFieldConfig[]) => void;
+};
+export const DerivedFields = (props: Props) => {
+  const { value, onChange } = props;
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  return (
+    <>
+      <h3 className="page-heading">Derived fields</h3>
+
+      <div className={styles.infoText}>
+        Derived fields can be used to extract new fields from the log message and create link from it's value.
```

Comment:
So this is not really true, it is not about extracting new fields from message field. But matching an existing field and adding a link to it?

Reply:
Yes sorry forgot to push the wording change.